src/unified_mocap_client.cpp

```diff
@@ -490,61 +543,79 @@ left │                          │ right )";
             }
             break;
         case CoordinateSystem::NED:
-            switch (this->long_edge)
+            switch (this->co_north)
             {
                 case ArenaDirection::RIGHT:
                     std::cout << R"( 
      │                          │
-     │   z  ⓧ → x              │
+     │   z  ⓧ → x               │
```

Comment:
NIT: align vertical lines

Also below

Reply:
works on my machine :sweat_smile: 
![image](https://github.com/tudelft/UnifiedOptitrackClients/assets/28341894/563ebec7-3966-481b-b541-e58985d9bc4c)

Reply:
I think, that's my fault. No need to handle it